src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+</head>
+<body>
+    
+</body>
+</html>
```

Comment:
You should add something to the body of your html file eg. `<p>Hello world</p>`

Reply:
@niicode Thank you for the feedback